deploy/charts/cert-manager/values.yaml

```
@@ -4,49 +4,49 @@
   #
   # For example:
   #  imagePullSecrets:
   #    - name: "image-pull-secret"
   imagePullSecrets: []
 
-  # Labels to apply to all resources
+  # Labels to apply to all resources.
   # Please note that this does not add labels to the resources created dynamically by the controllers.
   # For these resources, you have to add the labels in the template in the cert-manager custom resource:
-  # eg. podTemplate/ ingressTemplate in ACMEChallengeSolverHTTP01Ingress
-  #    ref: https://cert-manager.io/docs/reference/api-docs/#acme.cert-manager.io/v1.ACMEChallengeSolverHTTP01Ingress
-  # eg. secretTemplate in CertificateSpec
-  #    ref: https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec
+  # For example, podTemplate/ ingressTemplate in ACMEChallengeSolverHTTP01Ingress
+  # For more information, see the [cert-manager documentation](https://cert-manager.io/docs/reference/api-docs/#acme.cert-manager.io/v1.ACMEChallengeSolverHTTP01Ingress).
+  # For example, secretTemplate in CertificateSpec
+  # For more information, see the [cert-manager documentation](https://cert-manager.io/docs/reference/api-docs/#cert-manager.io/v1.CertificateSpec).
   commonLabels: {}
 
-  # The number of old ReplicaSets to retain to allow rollback (If not set, default Kubernetes value is set to 10)
+  # The number of old ReplicaSets to retain to allow rollback (if not set, the default Kubernetes value is set to 10).
   # +docs:property
   # revisionHistoryLimit: 1
 
-  # Optional priority class to be used for the cert-manager pods
+  # The optional priority class to be used for the cert-manager pods.
   priorityClassName: ""
 
   rbac:
-    # Create required ClusterRoles and ClusterRoleBindings for cert-manager
+    # Create required ClusterRoles and ClusterRoleBindings for cert-manager.
     create: true
-    # Aggregate ClusterRoles to Kubernetes default user-facing roles. Ref: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles
+    # Aggregate ClusterRoles to Kubernetes default user-facing roles. For more information, see [User-facing roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)
     aggregateClusterRoles: true
 
   podSecurityPolicy:
-    # Create PodSecurityPolicy for cert-manager
+    # Create PodSecurityPolicy for cert-manager.
     #
-    # NOTE: PodSecurityPolicy was deprecated in Kubernetes 1.21 and removed in 1.25
+    # Note that PodSecurityPolicy was deprecated in Kubernetes 1.21 and removed in Kubernetes 1.25.
     enabled: false
-    # Configure the PodSecurityPolicy to use AppArmor
+    # Configure the PodSecurityPolicy to use AppArmor.
     useAppArmor: true
 
-  # Set the verbosity of cert-manager. Range of 0 - 6 with 6 being the most verbose.
+  # Set the verbosity of cert-manager. A range of 0 - 6. with 6 being the most verbose.
```

suggestion: I think this should be a comma, not a `.`

```suggestion
  # Set the verbosity of cert-manager. A range of 0 - 6, with 6 being the most verbose.
```